lib/extensions.js

```diff
@@ -89,8 +89,9 @@ function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
          package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
```

Comment:
You can augment this with ['v', package.version, '/', sass.binaryName].join('')`